app/src/test/java/backgammon/BackgammonTests.java

```diff
@@ -61,6 +62,32 @@ public void canCalculateBarMove() {
   }
 
 
+  @Test
```

Comment:
This file has since been modified but even so you shouldn't have added your tests here.
The convention is to always make a new file for tests for each individual class.
Unit testing refers to testing each class separately as an individual unit.
You should have fixed this when I gave you the original feedback for your PR.

Reply:
I'll fix this too but make sure you follow this convention for next PR.